Client/Clique/Common/Models/Category.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Category.swift
+//  Clique
+//
+//  Created by Infinum on 28.11.2022..
+//
+
+import Foundation
+
+struct Category: Codable {
+    let category_id: Int
+    let category_name: String
+    let category_color: String
```

Comment:
```suggestion
    let id: Int
    let name: String
    let color: String
```

implement coding keys